cookbooks/continental_extension.prm

```diff
@@ -0,0 +1,254 @@
+#### Continental Extension Cookbook
+# This cookbook is based off numerous published studies, three of which are listed below.
+# For additional information, see these publications and references therein.
+#   1. Brune, S., Heine, C., Perez-Gussinye, M., and Sobolev, S.V. (2014), Nat. Comm., v.5, n.4014,
+#      Rift migration explains continental margin asymmetry and crustal hyperextension
+#   2. Huismans, R., and Beaumont, C. (2011), Nature, v.473, p.71-75.
+#      Depth-dependent extension, two-stage breakup and cratonic underplating at rifted margins
+#   3. Naliboff, J., and Buiter, S.H. (2015), Earth Planet. Sci. Lett., v.421, p.58-67,
+#      "Rift Reactivation and migration during multiphase extension"
+
+####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 5e6
+set Use years in output instead of seconds = true
+set Linear solver tolerance                = 1e-7
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-4
+set Max nonlinear iterations               = 10
+set Number of cheap Stokes solver steps    = 0
+set CFL number                             = 0.5
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
```

Comment:
Good eye! Instabilities can indeed develop near the surface with a Drucker-Prager rheology when there is no pressure constraint on the free surface.  Other FEM codes employing a free surface use a techniques to handle this, either by setting the surface elements pressure to 0, the lithostatic pressure or uniformly adding a constant value to the entire pressure field to ensure the minimum pressure is 0.  If the pressure is negative,  the `Drucker Prager` and `Visco Plastic` material models use a pressure value of 0 in the constitutive equations.

This type of pressure instability near the surface can have a noticeable affect in models where the pressures arising from deformation are much larger than the lithostatic pressure, such as the GeoMod experiments.  In crustal or lithospheric scale models, the effect is likely to be minimal at best (one would hope).

However, it is on my list to put one or more of these fixes in for ASPECT.  Currently, the `Pressure normalization` cannot be used with a free surface, but I don't think this would be too hard modify?  Adding a constant term to the entire pressure field to ensure a minimum pressure of 0 might be the easiest quick fix?